exercises/wordy/wordy_test.py

```diff
@@ -51,12 +51,14 @@ def test_multiple_division(self):
             calculate("What is -12 divided by 2 divided by -3?"), 2)
 
     def test_unknown_operation(self):
-        with self.assertRaisesWithMessage(ValueError):
+        with self.assertRaisesWithMessage(ValueError) as error:
```

Comment:
In this track, we don't test for actual exception message content, only that one is present. The reasoning for this is summarized well here: https://github.com/exercism/python/issues/1080#issuecomment-342013223

Reply:
Ok. Well should i check that the error message is not empty? Or is raising an error enough?

Reply:
`assertRaisesWithMessage` checks for a non-null, non-empty error message.